src/index.ts

```diff
@@ -16,539 +16,12 @@
  * Developed on behalf of: NYPL, Bokbasen AS (https://www.bokbasen.no), CAST (http://www.cast.org)
  * Licensed to: NYPL, Bokbasen AS and CAST under one or more contributor license agreements.
  */
+import D2Reader from "./reader";
```

Comment:
The contents of `/index.ts` were moved to `/reader.ts` so that `/index.ts` can be simply the API into the package, exporting all the things consumers need, not actually defining any implementation.